### PR TITLE
Fix:-Removed github text label and replaced it with Github icon

### LINKED ISSUE #403: Feature Request:- Replacing plain github label with the Github icon in the navBar
A clear and concise description of what the problem is.
Currently at the top Nav, the github link is just a plain label replacing it with the proper github icon seems an appropriate way.

**Describe the solution you'd like**
Adding a className to the item field in docusaurus config would solve this. we can use a background image externally and then use the icon.

**Describe alternatives you've considered**

**Approach to be followed (optional)**
A clear and concise description of approach to be followed.

**Additional context**

https://user-images.githubusercontent.com/72331432/224008882-01289675-5fed-4b08-ab94-2c3dc91a8e68.mp4

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Biki-das**
**What kind of change does this PR introduce?**

Replaced the Plain Github label link text with the Github Icon.

**Issue Number:**

Fixes  #403 

**Did you add tests for your changes?**

No

**Snapshots/Videos:**

https://user-images.githubusercontent.com/72331432/224105939-698abb86-5956-4358-b8e3-7c7cb3eb9087.mp4

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -177,15 +177,23 @@ h1.docTitle_node_modules-\@docusaurus-theme-classic-src-theme-DocItem- {
 
 .header-github-link:before {
     content: '';
-    width: 20px;
-    height: 20px;
+    width: 24px;
+    height: 24px;
     display: flex;
-    background: url('/img/github.svg') no-repeat;
```
Comment (palisadoes) [inline src/css/custom.css:183]: Why was this deleted? The image is there in the repository.

Comment (palisadoes) [reply src/css/custom.css:183]: Is this an SVG file that you have added? What advantage does this provide if any?

**Review by palisadoes — CHANGES_REQUESTED** [review]

File: src/css/custom.css (hunk 1 above)
Comment (Biki-das) [reply src/css/custom.css:183]: This is the official Github svg , commonly used by all docusaurus sites to implement the icon and is a compressed one,

Comment (palisadoes) [reply src/css/custom.css:183]: You have not answered the questions.

Comment (Biki-das) [reply src/css/custom.css:183]: No i dont see any advantage using this svg!

Comment (palisadoes) [reply src/css/custom.css:183]: Then why did you do this?

Comment (Biki-das) [reply src/css/custom.css:183]: I felt this could be a easier approach to use the external svg to link a background image , apologize for that , i can rebase to use the exisiting svg file.

Comment (palisadoes) [reply src/css/custom.css:183]: This is a design related issue, and if we use an external links for content then we have no control over elements of the design. Please resolve this.

Comment (Biki-das) [reply src/css/custom.css:183]: thanks for the feedback, I understood your feedback, resolving soon.

Comment (Biki-das) [reply src/css/custom.css:183]: @palisadoes need to create one more svg to have a darker one for the light mode, i hope that is fine, rest everything looks fine now
